lib/capybara/selenium/driver.rb

```diff
@@ -302,7 +302,11 @@ def clear_browser_state
   end
 
   def clear_browser_state_errors
-    @clear_browser_state_errors ||= [Selenium::WebDriver::Error::UnknownError]
+    @clear_browser_state_errors ||=
```

Comment:
If this needs changing for IE it should be done in the IE specialization, not in the general Selenium driver

Reply:
This is the code that was being called (So I'm guessing there's a super call somewhere, I'll track it down).